apps/web/public/static/locales/en/common.json

```
@@ -3742,5 +3745,19 @@
   "voice_id": "Voice ID",
   "use_voice": "Use Voice",
   "current_voice": "Current Voice",
+  "setup_incoming_agent": "Set up incoming agent",
+  "setup_incoming_agent_description": "Configure an AI agent to handle incoming calls on your phone number",
+  "connect_a_phone_number_first": "Connect a Phone Number first",
+  "setup_agent_for_incoming_calls": "Set up Agent for incoming calls",
+  "configure_agent_to_handle_incoming_calls": "Configure agent to handle incoming calls",
+  "incoming_calls": "Incoming Calls",
+  "outgoing_calls": "Outgoing Calls",
+  "inbound_agent_setup_success": "Inbound agent set up successful",
```

```suggestion
  "inbound_agent_setup_success": "Inbound agent setup successful",
```

It's a noun here so should be setup